src/main/java/co/rsk/bitcoinj/script/ErpFederationRedeemScriptParser.java

```diff
@@ -13,6 +13,8 @@
 public class ErpFederationRedeemScriptParser extends StandardRedeemScriptParser {
     private static final Logger logger = LoggerFactory.getLogger(ErpFederationRedeemScriptParser.class);
 
+    public static long MAX_CSV_VALUE = 105_120L; // 2 years in BTC blocks (considering 1 block every 10 minutes)
```

Comment:
This should be lower, most likely 65535 or less. The rationale is from BIP-68 and BIP-112.
 
![image](https://user-images.githubusercontent.com/3151940/180222433-9a9e50f3-f17a-413f-afaf-09e449cf70ff.png)

The CSV value has at most 4 bytes (well, 5 in the stack, for some special cases, but that can be ignored).
Of those 4 bytes, _only_ the first 16 bits are considered. 

More details provided to @marcos-iov and @josedahlquist  over Slack.

Reply:
Updated